tests/test_wave_01.py

```diff
@@ -60,13 +60,10 @@ def test_get_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
+    assert response_body == {"message": "Task 1 is not found."}
```

Comment:
Nice work with this test completion! This assertion is a great way to validate functionality is working as expected